docs/src/man/basics.md

```diff
@@ -0,0 +1,2002 @@
+
+```jldoctest dataframe
+julia> df = select(german, :Sex)
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+
+julia> df.Sex === german.Sex # copy
+false
+
+julia> df = select(german, :Sex, copycols=false)
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+
+julia> df.Sex === german.Sex # no-copy is performed
+true
+```
+
+To perform the selection operation in-place use `select!`:
+
+```jldoctest dataframe
+julia> german = copy(german_ref);
+
+julia> select!(german, Not(:Age));
+
+julia> german
+1000×9 DataFrame
+  Row │ id     Sex     Job    Housing  Saving accounts  Checking account  Cred ⋯
+      │ Int64  String  Int64  String   String           String            Int6 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0  male        2  own      NA               little                 ⋯
+    2 │     1  female      2  own      little           moderate
+    3 │     2  male        1  own      little           NA
+    4 │     3  male        2  free     little           little
+    5 │     4  male        2  free     little           little                 ⋯
+    6 │     5  male        1  free     NA               NA
+    7 │     6  male        2  own      quite rich       NA
+    8 │     7  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮       ⋮       ⋮            ⋮                ⋮               ⋱
+  994 │   993  male        3  own      little           little                 ⋯
+  995 │   994  male        2  own      NA               NA
+  996 │   995  female      1  own      little           NA
+  997 │   996  male        3  own      little           little
+  998 │   997  male        2  own      little           NA                     ⋯
+  999 │   998  male        2  free     little           little
+ 1000 │   999  male        2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+`transform` and `transform!` functions work identically to `select` and `select!` with the only difference that
+they retain all columns that are present in the source data frame and another difference is that `transform` and
+`transform!` always copy columns when column renaming transformation is passed.
+
+```jldoctest dataframe
+julia> german = copy(german_ref);
+
+julia> transform(german, :Age => maximum)
```

Comment:
the result of this operation is not seen

Reply:
Resolved now.